src/user-synchronizer/__main__.py

```diff
@@ -17,6 +17,7 @@
 
 # Read environment variables
 domain_offset_file = environ.get('DOMAIN_OFFSET_FILE', None)
+restfulapi_url = environ['RESTFULAPI_URL']
```

Comment:
I didn't find any more uses of `restfulapi_url` in this file except calls in `restfulapi.py`.

If this is just a restructure of moving all env vars to `__main__` instead of the actual module, could you please help move env retrieving in `logger.py` and `microsoft_graph.py` to `__main__` as well? This will prevent confusion in future code readers.

Reply:
I moved to __main__ because I want to be able to test restfulapi.py by calling it directly (added __name__ check there).

Ok, I think environment variable read should happen in main module, not library. I will move variables from those module to main as well.

Reply:
I'm a little confused about Fetch token part in `microsoft_graph.py`, the fetched token seems not used by any function. oauth will be configured by calling `fetch_token`?

Reply:
Yes, it will configure `self.token` as well as return it at the same time.